### ISSUE TITLE
Use with vpn

### ISSUE THREAD

**Issue by sam-gurdus** [martin-ger/esp_wifi_repeater#127]
Is it possible to set this up so that all Internet traffic going through it is sent over a vpn?

**Comment by sam-gurdus** [comment]
If not, does anyone know of an alternative? Thanks!

**Comment by martin-ger** [comment]
Actually, it would be possible to tunnel the uplink traffic through e.g. a TCP-link. But I am not aware of any VPN-client for the ESP or the lwip. Probably PPTP would be a good candidate. While it is not the strongest encryption, it is probably of the size that would be realistic for the ESP.

I wont have the time to start a VPN client project, but I there is one, I will look into integration with my router.

**Comment by sam-gurdus** [comment]
Yeah, for my use, I just want to use the vpn to bypass website blocks, so I suppose encryption isn't needed.

**Comment by yunfan** [comment]
@martin-ger how about n2n , p2p vpn

@Glenta  any documents showing how to use this with proxy? i could have a socks5 proxy
